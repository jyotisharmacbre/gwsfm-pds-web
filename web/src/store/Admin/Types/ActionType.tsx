export enum ActionType {
	GET_PROJECT_PARAMETERS_SUCCESS = 'GET_PROJECT_PARAMETERS_SUCCESS',
	GET_PROJECT_PARAMETERS_ERROR = 'GET_PROJECT_PARAMETERS_ERROR'
}
