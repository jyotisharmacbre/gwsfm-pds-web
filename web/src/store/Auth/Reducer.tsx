import { updateObject } from '../../helpers/utility-helper';
import { initialState } from './InitialState';
import { AuthenticationActions } from 'react-aad-msal';

const initializing = (oldState, action) => {
    return oldState;
};

const initialized = (oldState, action) => {
    return oldState;
};

const loginSuccess = (oldState, action) => {
    return oldState;
};

const loginFailed = (oldState, action) => {
    return oldState;
};
const loginError = (oldState, action) => {
    return oldState;
};
const clearError = (oldState, action) => {
    return oldState;
};
const logoutSuccess = (oldState, action) => {
    return updateObject(oldState, {
        token: updateObject(oldState.token, {
            idToken: null,
            expiration: null
        }),
        claims: updateObject(oldState.claims, {
            name: null,
            firstName: null,
            email: null
        })
    });
};
const acquiredIdTokenSuccess = (oldState, action) => {
    return updateObject(oldState, {
        token: updateObject(oldState.token, {
            idToken: action.payload.idToken.rawIdToken,
            expiration: action.payload.idToken.expiration
        }),
        claims: updateObject(oldState.claims, {
            name: action.payload.idToken.claims.name,
            firstName: action.payload.idToken.claims.given_name,
            email: action.payload.idToken.claims.email
        })
    });
};
const acquiredIdTokenError = (oldState, action) => {
    return oldState;
};
const acquiredAccessTokenSuccess = (oldState, action) => {
    return oldState;
};
const acquiredAccessTokenError = (oldState, action) => {
    return oldState;
};
const authenticatedStateChanged = (oldState, action) => {
    return oldState;
};

const authReducer = (oldState = initialState, action) => {
    switch (action.type) {
        case AuthenticationActions.Initializing:
            return initializing(oldState, action);
        case AuthenticationActions.Initialized:
            return initialized(oldState, action);
        case AuthenticationActions.LoginSuccess:
            return loginSuccess(oldState, action);
        case AuthenticationActions.LoginFailed:
            return loginFailed(oldState, action);
        case AuthenticationActions.LoginError:
            return loginError(oldState, action);
        case AuthenticationActions.ClearError:
            return clearError(oldState, action);
        case AuthenticationActions.LogoutSuccess:
            return logoutSuccess(oldState, action);
        case AuthenticationActions.AcquiredIdTokenSuccess:
            return acquiredIdTokenSuccess(oldState, action);
        case AuthenticationActions.AcquiredIdTokenError:
            return acquiredIdTokenError(oldState, action);
        case AuthenticationActions.AcquiredAccessTokenSuccess:
            return acquiredAccessTokenSuccess(oldState, action);
        case AuthenticationActions.AcquiredAccessTokenError:
            return acquiredAccessTokenError(oldState, action);
        case AuthenticationActions.AuthenticatedStateChanged:
            return authenticatedStateChanged(oldState, action);
        default:
            return oldState;
    }
};
export default authReducer;