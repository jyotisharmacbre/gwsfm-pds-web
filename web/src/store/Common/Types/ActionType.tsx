export enum ActionType {
  DEFAULT = 'DEFAULT'
}
