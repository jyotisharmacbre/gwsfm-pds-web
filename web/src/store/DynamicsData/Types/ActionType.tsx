export enum ActionType {
  DYNAMIC_CONTRACT_DATA_GET_SUCCESS = 'DYNAMIC_CONTRACT_DATA_GET_SUCCESS',
  DYNAMIC_CONTRACT_DATA_GET_ERROR = 'DYNAMIC_CONTRACT_DATA_GET_ERROR',
  DYNAMIC_COMPANY_DATA_GET_SUCCESS = 'DYNAMIC_COMPANY_DATA_GET_SUCCESS',
  DYNAMIC_COMPANY_DATA_GET_ERROR = 'DYNAMIC_COMPANY_DATA_GET_ERROR',
  DYNAMIC_OTHER_CONTRACT_SUCCESS = 'DYNAMIC_OTHER_CONTRACT_SUCCESS',
  DYNAMIC_OTHER_COMPANY_SUCCESS = 'DYNAMIC_OTHER_COMPANY_SUCCESS',
  DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS = 'DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS',
  DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR = 'DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR',
  DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS = 'DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS',
}
