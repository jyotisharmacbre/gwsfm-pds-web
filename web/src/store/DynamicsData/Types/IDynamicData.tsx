export interface IDynamicContractData {
  id: string;
  name: string;
}

export interface IDynamicCompanyData {
  id: string;
  name: string;
}

export interface IDynamicSubContractorData {
  id: string;
  name: string;
}

export interface IDynamicOther {
  id: string;
  label: string;
}
