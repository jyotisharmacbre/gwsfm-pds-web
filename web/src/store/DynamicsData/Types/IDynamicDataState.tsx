import {
  IDynamicCompanyData,
  IDynamicContractCustomerData,
  IDynamicOther,
  IDynamicSubContractorData,
  IDynamicsDivision,
  IDynamicBusinessUnits
} from './IDynamicData';

export interface IDynamicDataState {
  dynamicsCompany: Array<IDynamicCompanyData>;
  dynamicsListOfDivision: Array<IDynamicsDivision>;
  dynamicsContract: Array<IDynamicContractCustomerData>;
  error: string | null;
  dynamicsOtherContract: Array<IDynamicOther>;
  dynamicsOtherCompany: Array<IDynamicOther>;
  dynamicsSubcontractor: Array<IDynamicSubContractorData>;
  dynamicsOtherSubContractor: Array<IDynamicOther>;
  dynamicsListOfBusinessUnits:Array<IDynamicBusinessUnits>;
}
