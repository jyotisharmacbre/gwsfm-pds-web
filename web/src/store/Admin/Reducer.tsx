import { ActionType } from './Types/ActionType';
import { updateObject } from '../../helpers/utility-helper';
import { initialState } from './InitialState';

const getProjectParametersSuccess = (oldState, action) => {
	return updateObject(oldState, {
		adminDefaultValues: action.payload
	});
};

const getProjectParametersError = (oldState, action) => {
	return updateObject(oldState, {
		error: action.payload
	});
};

const adminReducer = (oldState = initialState, action) => {
	switch (action.type) {
		case ActionType.GET_PROJECT_PARAMETERS_SUCCESS:
			return getProjectParametersSuccess(oldState, action);
		case ActionType.GET_PROJECT_PARAMETERS_ERROR:
			return getProjectParametersError(oldState, action);
		default:
			return oldState;
	}
};

export default adminReducer;
