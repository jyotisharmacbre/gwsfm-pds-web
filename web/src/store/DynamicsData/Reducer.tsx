import { ActionType } from './Types/ActionType';
import { updateObject } from '../../helpers/utility-helper';
import { IDynamicDataState } from './Types/IDynamicDataState';

const initialState: IDynamicDataState = {
  dynamicsCompany: [],
  dynamicsContract: [],
  error: null,
  dynamicsOtherCompany: [],
  dynamicsOtherContract: [],
  dynamicsSubcontractor: [],
  dynamicsOtherSubContractor: [],
  dynamicsListOfDivision: [],
  dynamicsListOfBusinessUnits:[]
};

const getDynamicContractSuccess = (oldState, action) => {
  return updateObject(oldState, {
    error: null,
    dynamicsContract: action.payload
  });
};

const getDynamicContractError = (oldState, action) => {
  return updateObject(oldState, {
    error: action.payload
  });
};

const getDynamicCompanySuccess = (oldState, action) => {
  return updateObject(oldState, {
    error: null,
    dynamicsCompany: action.payload
  });
};

const getDynamicCompanyError = (oldState, action) => {
  return updateObject(oldState, {
    error: action.payload
  });
};

const getDynamicSubContractorSuccess = (oldState, action) => {
  return updateObject(oldState, {
    error: null,
    dynamicsSubcontractor: action.payload
  });
};

const getDynamicSubContractorError = (oldState, action) => {
  return updateObject(oldState, {
    error: action.payload
  });
};

const getDynamicContractOther = (oldState, action) => {
  return updateObject(oldState, {
    dynamicsOtherContract: action.payload
  });
};

const getDynamicCompanyOther = (oldState, action) => {
  return updateObject(oldState, {
    dynamicsOtherCompany: action.payload
  });
};

const getDynamicSubContractorOther = (oldState, action) => {
  return updateObject(oldState, {
    dynamicsOtherSubContractor: action.payload
  });
};
const getDynamicDivisionSuccess = (oldState, action) => {
  return updateObject(oldState, {
    dynamicsListOfDivision: action.payload
  });
};
const getDynamicDivisionError = (oldState, action) => {
  return updateObject(oldState, {
    error: action.payload
  });
};
const getDynamicBusinessUnitSuccess=(oldState,action)=>
{
  return updateObject(oldState,{dynamicsListOfBusinessUnits:action.payload})
}
const getDynamicBusinessUnitError=(oldState,action)=>
{
  return updateObject(oldState,{error:action.payload})
}
const dynamicDataReducer = (oldState = initialState, action) => {
  switch (action.type) {
    case ActionType.DYNAMIC_CONTRACT_DATA_GET_SUCCESS:
      return getDynamicContractSuccess(oldState, action);
    case ActionType.DYNAMIC_CONTRACT_DATA_GET_ERROR:
      return getDynamicContractError(oldState, action);
    case ActionType.DYNAMIC_COMPANY_DATA_GET_SUCCESS:
      return getDynamicCompanySuccess(oldState, action);
    case ActionType.DYNAMIC_COMPANY_DATA_GET_ERROR:
      return getDynamicCompanyError(oldState, action);
    case ActionType.DYNAMIC_OTHER_CONTRACT_SUCCESS:
      return getDynamicContractOther(oldState, action);
    case ActionType.DYNAMIC_OTHER_COMPANY_SUCCESS:
      return getDynamicCompanyOther(oldState, action);
    case ActionType.DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS:
      return getDynamicSubContractorSuccess(oldState, action);
    case ActionType.DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR:
      return getDynamicSubContractorError(oldState, action);
    case ActionType.DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS:
      return getDynamicSubContractorOther(oldState, action);
    case ActionType.DYNAMIC_DIVISION_DATA_GET_SUCCESS:
      return getDynamicDivisionSuccess(oldState, action);
    case ActionType.DYNAMIC_DIVISION_DATA_GET_ERROR:
      return getDynamicDivisionError(oldState, action);
    case ActionType.DYNAMIC_BUSINESSUNIT_SUCCESS:
      return getDynamicBusinessUnitSuccess(oldState, action);
    case ActionType.DYNAMIC_BUSINESSUNIT_ERROR:
      return getDynamicDivisionError(oldState, action);
    default:
      return oldState;
  }
};

export default dynamicDataReducer;
