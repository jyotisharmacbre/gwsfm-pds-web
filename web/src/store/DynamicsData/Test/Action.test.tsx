import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import * as actions from '../../rootActions';
import {ActionType} from '../../Common/Types/ActionType';
import EventType from '../../../enums/EventType';
import { initialState } from '../Reducer';
import nock from 'nock';
import { baseURL } from '../../../client/client';
import { getDynamicContractSuccess, getDynamicCompanySuccess, getDynamicDivisionError, getDynamicDivisionSuccess, getDynamicContractOtherSuccess, getDynamicSubContractorError, getDynamicCompanyOtherSuccess, getDynamicSubContractorSuccess, getDynamicSubContractorOtherSuccess, getDefaultSuccess, getDynamicBusinessUnitSuccess, getDynamicBusinessUnitError, getDynamicContractListSuccess, getDynamicContractListError } from '../Action';

nock(baseURL)
  .get('/api/ERPLookup/getDivision')
  .reply(200, {data: initialState});

  nock(baseURL)
  .get('/api/ERPLookup/getSubContractors/test?topCount=50')
  .reply(200, {data: initialState});

  nock(baseURL)
  .get('/api/ERPLookup/getCompanies/test?topCount=50')
  .reply(200, {data: initialState});

  nock(baseURL)
  .get('/api/ERPLookup/getContractsAndCustomers/test?topCount=50')
  .reply(200, {data: initialState});

  nock(baseURL)
  .post('/api/ERPLookup/getContractsAndCustomersList')
  .reply(200, {data: initialState});



const mockStore = configureStore([ thunk ]);
let reduxStore;
const setUpStore = (initialState) => {
	reduxStore = mockStore({
		project: initialState
	});
};
describe('Dynamics data action tests', () => {
	beforeEach(() => {
		setUpStore(initialState);
		reduxStore.clearActions();
	});
   
	it('should dispatch the getDynamicDivisionSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getListOfDivision());
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicContractListSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getContractDetailsByIds(['test']));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicSubContractorData', () => {		
		const fun = jest.fn();
		setUpStore(initialState);
		actions.getDynamicSubContractorData('test', fun, fun);
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicContractOtherSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicOther(initialState, 'contractorId'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicCompanyOtherSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicOther(initialState, 'companyId'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicSubContractorOtherSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicOther(initialState, 'subcontractorId'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDefaultSuccess', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicOther(initialState, 'anythingelse'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicCompanyData', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicCompanyData('test'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should dispatch the getDynamicContractData', () => {		
		setUpStore(initialState);
		reduxStore.dispatch(actions.getDynamicCompanyData('test'));
		expect(reduxStore.getActions()).toMatchSnapshot();
	});

	it('should return getDynamicContractSuccess', () => {				
		expect(getDynamicContractSuccess({data: 'test'})).toMatchSnapshot();
	});

	it('should return getDynamicContractError', () => {				
		expect(getDynamicContractSuccess({error: 'test'})).toMatchSnapshot();
	});

	
	it('should return getDynamicCompanySuccess', () => {				
		expect(getDynamicCompanySuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicDivisionError', () => {				
		expect(getDynamicDivisionError({error: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicDivisionSuccess', () => {				
		expect(getDynamicDivisionSuccess({data: 'test'})).toMatchSnapshot();
	});

	it('should return getDynamicCompanyError', () => {				
		expect(getDynamicDivisionError({error: 'test'})).toMatchSnapshot();
	});

	it('should return getDynamicContractOtherSuccess', () => {				
		expect(getDynamicContractOtherSuccess({data: 'test'})).toMatchSnapshot();
	});

	it('should return getDynamicCompanyOtherSuccess', () => {				
		expect(getDynamicCompanyOtherSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicSubContractorSuccess', () => {				
		expect(getDynamicSubContractorSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicSubContractorError', () => {				
		expect(getDynamicSubContractorError({error: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicSubContractorOtherSuccess', () => {				
		expect(getDynamicSubContractorOtherSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDefaultSuccess', () => {				
		expect(getDefaultSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicBusinessUnitSuccess', () => {				
		expect(getDynamicBusinessUnitSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicBusinessUnitError', () => {				
		expect(getDynamicBusinessUnitError({error: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicContractListSuccess', () => {				
		expect(getDynamicContractListSuccess({data: 'test'})).toMatchSnapshot();
	});
	it('should return getDynamicContractListError', () => {				
		expect(getDynamicContractListError({error: 'test'})).toMatchSnapshot();
	});
});
