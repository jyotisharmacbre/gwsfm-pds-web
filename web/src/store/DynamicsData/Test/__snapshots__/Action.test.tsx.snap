// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dynamics data action tests should dispatch the getDefaultSuccess 1`] = `
Array [
  Object {
    "type": "DYNAMIC_OTHER_DEFAULT_SUCCESS",
  },
]
`;

exports[`Dynamics data action tests should dispatch the getDynamicCompanyData 1`] = `Array []`;

exports[`Dynamics data action tests should dispatch the getDynamicCompanyOtherSuccess 1`] = `
Array [
  Object {
    "payload": Object {
      "dynamicsCompany": Array [],
      "dynamicsContract": Array [],
      "dynamicsListOfBusinessUnits": Array [],
      "dynamicsListOfDivision": Array [],
      "dynamicsOtherCompany": Array [],
      "dynamicsOtherContract": Array [],
      "dynamicsOtherSubContractor": Array [],
      "dynamicsSubcontractor": Array [],
      "error": null,
    },
    "type": "DYNAMIC_OTHER_COMPANY_SUCCESS",
  },
]
`;

exports[`Dynamics data action tests should dispatch the getDynamicContractData 1`] = `Array []`;

exports[`Dynamics data action tests should dispatch the getDynamicContractListSuccess 1`] = `Array []`;

exports[`Dynamics data action tests should dispatch the getDynamicContractOtherSuccess 1`] = `
Array [
  Object {
    "payload": Object {
      "dynamicsCompany": Array [],
      "dynamicsContract": Array [],
      "dynamicsListOfBusinessUnits": Array [],
      "dynamicsListOfDivision": Array [],
      "dynamicsOtherCompany": Array [],
      "dynamicsOtherContract": Array [],
      "dynamicsOtherSubContractor": Array [],
      "dynamicsSubcontractor": Array [],
      "error": null,
    },
    "type": "DYNAMIC_OTHER_CONTRACT_SUCCESS",
  },
]
`;

exports[`Dynamics data action tests should dispatch the getDynamicDivisionSuccess 1`] = `Array []`;

exports[`Dynamics data action tests should dispatch the getDynamicSubContractorData 1`] = `Array []`;

exports[`Dynamics data action tests should dispatch the getDynamicSubContractorOtherSuccess 1`] = `
Array [
  Object {
    "payload": Object {
      "dynamicsCompany": Array [],
      "dynamicsContract": Array [],
      "dynamicsListOfBusinessUnits": Array [],
      "dynamicsListOfDivision": Array [],
      "dynamicsOtherCompany": Array [],
      "dynamicsOtherContract": Array [],
      "dynamicsOtherSubContractor": Array [],
      "dynamicsSubcontractor": Array [],
      "error": null,
    },
    "type": "DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS",
  },
]
`;

exports[`Dynamics data action tests should return getDefaultSuccess 1`] = `
Object {
  "type": "DYNAMIC_OTHER_DEFAULT_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicBusinessUnitError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_BUSINESSUNIT_ERROR",
}
`;

exports[`Dynamics data action tests should return getDynamicBusinessUnitSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_BUSINESSUNIT_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicCompanyError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_COMPANY_DATA_GET_ERROR",
}
`;

exports[`Dynamics data action tests should return getDynamicCompanyOtherSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_OTHER_COMPANY_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicCompanySuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_COMPANY_DATA_GET_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicContractError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_CONTRACT_DATA_GET_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicContractListError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_CONTRACTLIST_ERROR",
}
`;

exports[`Dynamics data action tests should return getDynamicContractListSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_CONTRACTLIST_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicContractOtherSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_OTHER_CONTRACT_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicContractSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_CONTRACT_DATA_GET_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicDivisionError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_COMPANY_DATA_GET_ERROR",
}
`;

exports[`Dynamics data action tests should return getDynamicDivisionSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_DIVISION_DATA_GET_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicSubContractorError 1`] = `
Object {
  "payload": Object {
    "error": "test",
  },
  "type": "DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR",
}
`;

exports[`Dynamics data action tests should return getDynamicSubContractorOtherSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS",
}
`;

exports[`Dynamics data action tests should return getDynamicSubContractorSuccess 1`] = `
Object {
  "payload": Object {
    "data": "test",
  },
  "type": "DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS",
}
`;
